### PR TITLE
Clean docker environment after a build

### PR DESCRIPTION
**Pull request by lhanekam**
**JIRA Tickets:**

[AB2D-2813](https://jira.cms.gov/browse/AB2D-2813) - Clean docker environment on agent after a build

NOTE: The description has been updated in Jira ticket to explain why this is needed.

***Related Tickets***

N/A
 
### What Does This PR Do?

Cleans the docker environment on an agent after a build by doing the following:

1. Deletes all docker volumes

2. Deletes all stopped containers

3. Deletes all images except the following

   - postgres:11

   - testcontainers/ryuk:0.3.0

   - testcontainersofficial/ryuk:0.3.0

   - maven:3-jdk-13

   - openjdk:13

   - alpine/socat:1.7.3.4-r0

   - alpine/socat:latest

   - quay.io/testcontainers/ryuk:0.2.3

4. Deletes all docker networks except the following

   - bridge

   - host

   - none

### What Should Reviewers Watch For?

N/A

### Usage/Deployment Instructions

N/A

### Impacted External Components

N/A

### Database Changes

N/A

### Limitations

N/A

### Security Implications

- [ ] This PR adds new software dependencies
- [ ] This PR modifies or invalidates our security controls
- [ ] This PR stores or transmits data that was not stored or transmitted before

- [ ] This PR requires additional review of its security implications for other reasons

### Submitter Checklist

- [X] This PR includes any required documentation changes, (`README`, changelog / release notes, website).
- [X] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments.
- [X] Code checked for PHI/PII exposure

### CODE REVIEW

**Review by nflearl — DISMISSED** [review]

**Review by wnyffenegger — APPROVED** [review]

**Review by wnyffenegger — CHANGES_REQUESTED** [review]
CI build is causing issues on post build changes

File: e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java [hunk 1]
```diff
@@ -169,9 +175,9 @@ private void loadDockerComposeContainers(int apiPort) {
                 .withScaledService("worker", 2)
                 .withScaledService("api", 1)
                 .withExposedService("api", DEFAULT_API_PORT, new HostPortWaitStrategy()
-                    .withStartupTimeout(Duration.of(200, SECONDS)));
-//                .withLogConsumer("worker", new Slf4jLogConsumer(log)) // Use to debug, for now there's too much log data
-//                .withLogConsumer("api", new Slf4jLogConsumer(log));
+                    .withStartupTimeout(Duration.of(200, SECONDS)))
+                .withLogConsumer("worker", new Slf4jLogConsumer(workerLogger)) // Use to debug, for now there's too much log data
+                .withLogConsumer("api", new Slf4jLogConsumer(apiLogger));
```
Comment (nflearl) [inline e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java:180]: Might want to recomment these out.

**Review by nflearl — DISMISSED** [review]
Might want to re-comment the logging back out.

**Review by wnyffenegger — DISMISSED** [review]
I would like to leave the loggers in so we know when an issue has been caused by liquibase and when an issue has been caused by something else in the e2e tests.

File: e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java (hunk 1 above)
Comment (nflearl) [reply e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java:180]: Quick chat, Will wants to make these semi-permanent.  I'm fine with that.

Comment needs to be made more accurate or deleted.

**Review by nflearl — CHANGES_REQUESTED** [review]

**Review by nflearl — APPROVED** [review]

**Review by lailasharshar — APPROVED** [review]

**Pull request merged**